server/src/api/modules/competitions/competition.service.js

```diff
@@ -272,12 +272,16 @@ async function create(title, metric, startsAt, endsAt, groupId, participants) {
 
   // Check if every username in the list is valid
   if (participants && participants.length > 0) {
-    for (let i = 0; i < participants.length; i += 1) {
-      if (!playerService.isValidUsername(participants[i])) {
-        throw new BadRequestError(`Invalid player username: ${participants[i]}`);
-      }
+    const usernamesCheck = await playerService.isValidUsernames(participants);
```

Comment:
Maybe this variable could be renamed to "invalidUsernames"?

**Also: isValidUsernames is not an async function so it does not need the "await" before calling it.**

Reply:
I must've been late night reading because I thought I read async on the player service function. :) I'll remove the await.